homeassistant/components/plugwise/strings.json

```
@@ -259,7 +259,7 @@
         "name": "DHW comfort mode"
       },
       "lock": {
-        "name": "[%key:component::lock::entity_component::_::name%]"
+        "name": "Lock"
```

```suggestion
        "name": "[%key:component::lock::title%]"
```

Lokalise only supports one level references